document_merge_service/api/management/commands/dms_encrypt_templates.py

```diff
@@ -0,0 +1,49 @@
+from django.conf import settings
+from django.core.files.storage import storages
+from django.core.management.base import BaseCommand
+from tqdm import tqdm
+
+from document_merge_service.api.models import Template
+
+
+class Command(BaseCommand):
+    help = "Swaps plain text template content to encrypted content"
+
+    def handle(self, *args, **options):
+        if not settings.DMS_ENABLE_AT_REST_ENCRYPTION:
+            return self.stdout.write(
+                self.style.WARNING(
+                    "Encryption is not enabled. Skipping encryption of templates."
+                )
+            )
+
+        failed_templates = []
+
+        # flip between default and encrypted storage to have the correct parameters in the requests
+        encrypted_storage = storages.create_storage(settings.STORAGES["default"])
+        normal_storage_setting = settings.STORAGES["default"]
+        del normal_storage_setting["OPTIONS"]["object_parameters"]
```

Comment:
That will throw a key error, if `OPTIONS` or `object_parameters` don't exist.

Either only `del` if the item exists or raise `ConfigurationError` if that's required.

Reply:
Added check